### PR TITLE
ISSUE #4893 - New toolbar colours

### LINKED ISSUE #4893: Improve toolbar states.
### Description
In the new toolbar the active/disabled/inactive states are too subtle so users have a hard time discerning  the actual state of the button.

### Goals
- [ ] As a user, I want to be able to easily distinguish what tools are available, and what tools are currently enabled.

### Tasks
- [ ] update the ui using the design

### Related Resources
[MockUp](https://www.figma.com/file/yzsLESycxJwhzw4bqLOS0C/3D-Repo-UI-Components?type=design&node-id=123-6119&mode=design&t=bSunC0cTe5NZrYbz-0)

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #4893

#### Description
Toolbar uses different colours to make it easier identifying what is active/disabled/selected/nothing.

Because the perspective icons looked weird with the new design, their opacity was dropped to 50% directly in the svg.

Some colours in the mockup were created using transparencies. we don't want to do that, instead, the closer solid colour in our palette was used in those scenarios

#### Test cases
Open up the viewer and check the toolbar buttons.

The icons in a button with a submenu should change its colour when the submenu is expanded. The same should happen to the submenu indicator (the little triangle on the top right).
This also applies to the markup toolbar to keep the design consistent

### CODE REVIEW

**Review by carmenfan — CHANGES_REQUESTED** [review]
Much clearer now,  only one thing:

Could we change it so the clear selection text change to primary main on hover?

**Comment by Amantini1997** [comment]
> Much clearer now, only one thing:
> 
> Could we change it so the clear selection text change to primary main on hover?  [  ](https://private-user-images.githubusercontent.com/11945337/324037269-b1495f7f-6265-4c7b-ace7-ec91226c3394.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1NDU2MTAsIm5iZiI6MTcxMzU0NTMxMCwicGF0aCI6Ii8xMTk0NTMzNy8zMjQwMzcyNjktYjE0OTVmN2YtNjI2NS00YzdiLWFjZTctZWM5MTIyNmMzMzk0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDE2NDgzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwYTI0MzM1NjFlOTQzOGU0MjJlNzE0YWFmOGRmOGIzMzE5ZGU5Yjc2Y2ExZDQ1NWVjZDE5YTkxNWQ4ZDE1MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1_wtSXdKswN7n8lmxyaLjmKGGoySL84vIFgFQMYnW8A) [ ](https://private-user-images.githubusercontent.com/11945337/324037269-b1495f7f-6265-4c7b-ace7-ec91226c3394.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1NDU2MTAsIm5iZiI6MTcxMzU0NTMxMCwicGF0aCI6Ii8xMTk0NTMzNy8zMjQwMzcyNjktYjE0OTVmN2YtNjI2NS00YzdiLWFjZTctZWM5MTIyNmMzMzk0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDE2NDgzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwYTI0MzM1NjFlOTQzOGU0MjJlNzE0YWFmOGRmOGIzMzE5ZGU5Yjc2Y2ExZDQ1NWVjZDE5YTkxNWQ4ZDE1MzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1_wtSXdKswN7n8lmxyaLjmKGGoySL84vIFgFQMYnW8A)

Should the background colour also change?

**Review by carmenfan — DISMISSED** [review]

File: frontend/assets/icons/viewer/focus.svg.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default ({ className }: IProps) => (
 	<svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
 		<g className="move-cube-o">
 			<path
-				className="highlight"
+				className="primary"
```
Comment (sanmont3drepo) [inline frontend/assets/icons/viewer/focus.svg.tsx:25]: If we are now just using one colour get rid of classname entirely

File: frontend/assets/icons/viewer/align.svg.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ type IProps = {
...
-		<path className="highlight" id="Rectangle 3623" d="M6.5588 9.46249C5.95347 9.113 5.74606 8.33895 6.09555 7.73361L9.59292 1.67599C9.94242 1.07065 10.7165 0.863249 11.3218 1.21274L17.3794 4.71011C17.9848 5.0596 18.1922 5.83365 17.8427 6.43899L14.3453 12.4966C13.9958 13.1019 13.2218 13.3094 12.6164 12.9599L6.5588 9.46249Z" fill="currentColor"/>
+		<path className="primary" id="Rectangle 3623" d="M6.5588 9.46249C5.95347 9.113 5.74606 8.33895 6.09555 7.73361L9.59292 1.67599C9.94242 1.07065 10.7165 0.863249 11.3218 1.21274L17.3794 4.71011C17.9848 5.0596 18.1922 5.83365 17.8427 6.43899L14.3453 12.4966C13.9958 13.1019 13.2218 13.3094 12.6164 12.9599L6.5588 9.46249Z" fill="currentColor"/>
```
Comment (sanmont3drepo) [inline frontend/assets/icons/viewer/align.svg.tsx:25]: If we are now just using one colour get rid of classname entirely as the code implies more complexity by having to check a non existing class.   Same for all svgs.

File: frontend/src/v5/ui/components/shared/modalsDispatcher/templates/imagesModal/imageMarkup/markupToolbar/buttons/strokeWidthButton/strokeWidthButton.component.tsx [hunk 3]
```diff
@@ -25,9 +25,9 @@ import { ToolbarSelect } from '@controls/toolbarSelect/toolbarSelect.component';
 import { Tooltip } from '@mui/material';
 
 const VALUE_TO_WIDTH = invert(STROKE_WIDTH);
-const Icon = ({ value, title }) => (
+const Icon = ({ value, title, expanded }) => (
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/components/shared/modalsDispatcher/templates/imagesModal/imageMarkup/markupToolbar/buttons/strokeWidthButton/strokeWidthButton.component.tsx:28]: I don't know if this makes sense, but do you think that expanded could be part of a context?. I see a lot of expanded passing around

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/components/shared/modalsDispatcher/templates/imagesModal/imageMarkup/markupToolbar/buttons/strokeWidthButton/strokeWidthButton.component.tsx (hunk 3 above)
Comment (Amantini1997) [reply frontend/src/v5/ui/components/shared/modalsDispatcher/templates/imagesModal/imageMarkup/markupToolbar/buttons/strokeWidthButton/strokeWidthButton.component.tsx:28]: I gave it a try and it didn't make the code easier to read/maintain

**Review by sanmont3drepo — APPROVED** [review]

**Review by carmenfan — APPROVED** [review]

**Pull request merged**